packages/dart/sshnp_flutter/pubspec.yaml

```diff
@@ -77,3 +78,13 @@ flutter_launcher_icons:
   macos:
     generate: true
     image_path: "assets/images/app_logo.png"
+
+msix_config:
+  display_name: "SSH No Ports Desktop"
+  publisher_display_name: "Atsign"
+  identity_name: "TheCompany.SSHNoPortsDesktop"
```

Comment:
Should we change the identity name from "TheCompany" to "Atsign.SSHNoPortsDesktop" ? Or is it not relevant / important?

Reply:
It's for internal use and is not shown on windows stores.
We can keep it as it is.

Reply:
> It's for internal use and is not shown on windows stores. We can keep it as it is.

I think it would be better to change it unless that will cause a problem

Reply:
It can not be changed even if we update organisation name, identity_name will remain same.